assessments/projects/cni/self-assessment.md

```diff
@@ -0,0 +1,308 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+# Self-assessment outline
```

Comment:
```suggestion
```

Please go back to ensure that no template text like this is included here or in the rest of the document